cli.php

```diff
@@ -35,7 +37,10 @@
 // Create the CLI command instance and add to WP CLI.
 $plugin_command = new Plugin_Check_Command( $context );
 WP_CLI::add_command( 'plugin', $plugin_command );
-
+$plugin_categories_command = new Plugin_Check_Categories_Command( $context );
+WP_CLI::add_command( 'plugin', $plugin_categories_command );
+$plugin_checks_command = new Plugin_Check_Checks_Command( $context );
+WP_CLI::add_command( 'plugin', $plugin_checks_command );
```

Comment:
+1 for these proposed names. My only suggestion would be to stick to `plugin` over `plugin-check` as a base group name:
* It's more intuitive to use. `wp plugin-check check` or `wp plugin-check list-checks` feels more cumbersome to me.
* All our commands are related to plugins, so they fit well under the `plugin` group.
* While it can be a good practice to group under a plugin specific group, this plugin is a WordPress project, so it makes sense to go for convenience here, as we'll certainly not abuse the usage of this regular CLI group.

Reply:
> My only suggestion would be to stick to `plugin` over `plugin-check` as a base group name:

Then I don't think we can keep `wp plugin check <pluginslug>` working, as it becomes a group instead (but I'd need to verify first)

So are you suggesting a setup like this:

* `wp plugin check list-checks`
* `wp plugin check list-categories`
* `wp plugin check check-plugin`

?

Reply:
No, what I mean is:
* `wp plugin check`
* `wp plugin list-checks`
* `wp plugin list-check-categories`

Reply:
Hmm then the subcommands are a bit disconnected, it‘s not clear that the last 2 belong to Plugin Check.

Reply:
Why would the last 2 be different? They all would be in the `plugin` group and include the word "check" to clarify that they relate to plugin _check_ functionality.

Reply:
Just because they include the word "check" doesn't make it clear enough that all these three commands belong together IMHO. That's why WP-CLI commands are typically grouped together as I mentioned earlier.

I guess none of the possible options are super pleasing so let's just start with that, and then when we refactor things in #153 we can revisit.